### ISSUE TITLE
Oops while pasting a long command into VSCode's PowerShell terminal

### ISSUE THREAD

**Issue by Reilithion** [PowerShell/PSReadLine#2968]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/PSReadLine)
- [X] Search the existing issues, especially the pinned issues.

### Exception report

```console
Oops, something went wrong.  Please report this bug with the details below.
Report on GitHub: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
Last 200 Keys:
 0 \ l i b Space - L Space n a t i v e = C : \ U s e r s \ L u c a s \ D o c u m e n t s \ s h a r e \ l u c a s \ p r o j e c t s \ s i m p l e - d i s c o r d - b 
o t \ t a r g e t \ d e b u g \ b u i l d \ r i n g - 3 8 2 c 7 6 8 f e 6 c 3 8 d 8 e \ o u t Space - L Space n a t i v e = C : \ U s e r s \ L u c a s \ D o c u m e n t s \ s h a r e \ l u c a s \ p r o j e c t s \ s i m p l e - d i s c o r d - b o t

Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
Parameter name: top
Actual value was -5.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   at Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   at Microsoft.PowerShell.PSConsoleReadLine.SelfInsert(Nullable`1 key, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------
```

### Screenshot

![image](https://user-images.githubusercontent.com/38637/140903666-70684df9-8682-4a91-9a4e-fd202479a4ff.png)

### Environment data

```console
PS Version: 5.1.19041.1237
PS HostName: ConsoleHost
PSReadLine Version: 2.0.0-beta2
PSReadLine EditMode: Windows
OS: 10.0.19041.1 (WinBuild.160101.0800)
BufferWidth: 165
BufferHeight: 34
```

### Steps to reproduce

Not entirely certain, but I pasted this text into VSCode's terminal window and got the exception. I've placed it in a code block. It was one line (no newline characters as far as I know).

```
rustc --crate-name simple_discord_bot --edition=2021 src\main.rs --error-format=json --json=diagnostic-rendered-ansi --crate-type bin --emit=dep-info,link -C embed-bitcode=no -C debuginfo=2 -C metadata=1a7677da8306a154 -C extra-filename=-1a7677da8306a154 --out-dir C:\Users\Lucas\Documents\share\lucas\projects\simple-discord-bot\target\debug\deps -C incremental=C:\Users\Lucas\Documents\share\lucas\projects\simple-discord-bot\target\debug\incremental -L dependency=C:\Users\Lucas\Documents\share\lucas\projects\simple-discord-bot\target\debug\deps --extern r2d2=C:\Users\Lucas\Documents\share\lucas\projects\simple-discord-bot\target\debug\deps\libr2d2-6492463cdb7bf82a.rlib --extern r2d2_sqlite=C:\Users\Lucas\Documents\share\lucas\projects\simple-discord-bot\target\debug\deps\libr2d2_sqlite-e4b1cf8d2a0418b6.rlib --extern rusqlite=C:\Users\Lucas\Documents\share\lucas\projects\simple-discord-bot\target\debug\deps\librusqlite-2abfb9c5be4e2dc2.rlib --extern serenity=C:\Users\Lucas\Documents\share\lucas\projects\simple-discord-bot\target\debug\deps\libserenity-8d722d249040f182.rlib --extern tokio=C:\Users\Lucas\Documents\share\lucas\projects\simple-discord-bot\target\debug\deps\libtokio-7402c0a28426458e.rlib --extern tracing=C:\Users\Lucas\Documents\share\lucas\projects\simple-discord-bot\target\debug\deps\libtracing-a46b1c284973cef3.rlib --extern tracing_subscriber=C:\Users\Lucas\Documents\share\lucas\projects\simple-discord-bot\target\debug\deps\libtracing_subscriber-20e1a8cefefa7e92.rlib -L native=C:\Users\Lucas\.cargo\registry\src\github.com-1ecc6299db9ec823\winapi-x86_64-pc-windows-gnu-0.4.0\lib -L native=C:\Users\Lucas\Documents\share\lucas\projects\simple-discord-bot\target\debug\build\ring-382c768fe6c38d8e\out -L native=C:\Users\Lucas\Documents\share\lucas\projects\simple-discord-bot
```

### Expected behavior

Clipboard contents should have been pasted into VSCode's terminal.

### Actual behavior

Exception occurred.

**Comment by iSazonov** [comment]
Please update latest Beta4 version.

**Comment by daxian-dbw** [comment]
This issue was already fixed (see #1306). Please upgrade to the [2.1.0 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.1.0) from PowerShell Gallery.
See the [upgrading section](https://github.com/PowerShell/PSReadLine#upgrading) for instructions. Please let us know if you run into the same issue with the latest version.
